module/Database/src/Form/Address.php

```diff
@@ -120,6 +126,11 @@ public function getInputFilterSpecification(): array
             ],
```

Comment:
https://github.com/GEWIS/gewisdb/blob/bd5050acf723dfc98ad293a1c5128be1b7fabc4d/module/Database/src/Form/Address.php#L112-L120

Perhaps we want to prohibit spaces at the end here as well? We could just remove spaces on either side, but also apply a regex. 
Complicated addresses: `Blauwe Loper 1 0001`, `De Lampendriessen 31-1417`, `Stationsweg 19-001`, `High Tech Campus 11a`, `Europalaan 1a-1` (all real)

```diff
- 'options' => ['pattern' => '/^[0-9]+[a-zA-Z]*/'],
+ 'options' => [
+     'pattern' => '/^([0-9a-zA-Z][\-\s]?)*[0-9a-zA-Z]$/',
+     'messages' => [
+         Regex::NOT_MATCH => $this->translator->translate(
+             'Only alphanumeric characters, spaces and dashes are allowed.',
+         ),
+     ],
+ ],
```

Reply:
I saw this one on SO:

```
^[1-9]\d*(?: ?(?:[a-z]|[/-] ?\d+[a-z]?))?$
```

But we should address house numbers in GH-438.